examples/1_Simple/tracing_fieldline.py

```diff
@@ -56,14 +56,14 @@
 ntor = 5
 stellsym = True
 nfp = 3
-phis = np.linspace(0, 1, nfp*2*ntor+1, endpoint=False)
-thetas = np.linspace(0, 1, 2*mpol+1, endpoint=False)
+phis = np.linspace(0, 1, 64, endpoint=True)
+thetas = np.linspace(0, 1, 24, endpoint=True)
```

Comment:
Usually we have `endpoint=False` for the quadpoints. Could you add a comment here to explain why we need `endpoint=True` in this scenario?

Reply:
And should `endpoint=True` be used also in the analogous line of `tracing_particles.py`?

Reply:
good catch, this was a left over from me debugging something. undid this.